lib/fluent/config/element.rb

```diff
@@ -73,7 +73,7 @@ def add_element(name, arg = '')
 
       def inspect
         attrs = super
-        "name:#{@name}, arg:#{@arg}, " + attrs + ", " + @elements.inspect
+        "<name:#{@name}, arg:#{@arg}, attrs:" + attrs + ", elements:" + @elements.inspect + ">"
```

Comment:
```suggestion
        "<name:#{@name}, arg:#{@arg}, attrs:#{attrs}, elements:#{@elements.inspect}>"
```

Thanks!
This would be clearer.
I have confirmed this does not change the output format.

Reply:
Thanks!